### PR TITLE
#AC-7583::Stores->Configuration page errors when enabled Fastly

### PR DESCRIPTION
**Pull request by glo05363**
updated zend_http with laminas_http

### CODE REVIEW

**Comment by glo05363** [comment]
@vvuksan please review and merge this PR.

**Comment by chernenm** [comment]
I would like to provide more details about the changes in this PR:
Currently when run Adobe Commerce 2.4.6 on Cloud with Fastly 1.2.194 enabled, the Admin configuration page fails with the following errors:
report.CRITICAL: Error: Class "Zend_Http_Client" not found in /app/vendor/fastly/magento2/Model/Notification.php:159
report.CRITICAL: Error: Class "Zend_Http_Response" not found in /app/vendor/fastly/magento2/Model/Statistic.php:596
 
The root cause of the issue is that in scope of Adobe Commerce 2.4.6 we upgraded from Zend_Http library to Laminas\Http\Client and Laminas\Http\Request, which are not compatible with  [https://github.com/fastly/fastly-magento2/commits/master,](https://github.com/fastly/fastly-magento2/commits/master)

Ex.
https://github.com/fastly/fastly-magento2/blob/master/Model/Notification.php
https://github.com/fastly/fastly-magento2/blob/master/Model/Statistic.php

This pull request is intended to updated this dependency for Fastly.

File: Model/Notification.php [hunk 1]
```diff
@@ -156,15 +158,15 @@ public function getLastVersion()
...
             $responseBody = $client->read();
             $client->close();
 
-            $responseCode = \Zend_Http_Response::extractCode($responseBody);
+            $responseCode = $this->extractCodeFromResponse($responseBody);
             if ($responseCode !== 200) {
                 return false;
             }
-            $body = \Zend_Http_Response::extractBody($responseBody);
+            $body = Response::getBody($responseBody);
```
Comment (dpotkoc) [inline Model/Notification.php:169]: getBody is not static method, it causes error.

Comment (dpotkoc) [reply Model/Notification.php:169]: with this PR we can't open fastly config in administration (system section)

Comment (glo05363) [reply Model/Notification.php]: @dpotkoc fixed the issue.

**Comment by vvuksan** [comment]
This pull request will be superseded by #606. Closing.

**Pull request closed without merging**